pkg/controller/config/agent.go

```diff
@@ -41,43 +41,47 @@ func (ca *Agent) Start(configLocation string) error {
 		return err
 	}
 	ca.Set(c)
+	watcher, err := fsnotify.NewWatcher()
+	if err != nil {
+		logrus.WithError(err).Error("Error when creating watcher.")
```

Comment:
IMO then it does not make sense to continue.

Reply:
Done. `Fital` now.